server/bin/pbench-cull-unpacked-tarballs.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+"""Pbench Cull Unpacked Tar Balls
+
+The maximum age of an unpacked tar ball is provided via the "unpacked-age"
+variable in the [pbench-server] configuration setting.  By default we only
+keep around unpacked tar balls for 30 days.
+
+An unpacked tar ball directory can be kept around longer than the maximum
+configured age by creating a ".__pbench_keep__" file in the top level of that
+directory.
+
+The culling of unpacked tar balls occurrs once a day. Each unpacked tar ball
+found in the ${INCOMING} directory hierarchy is checked against the configured
+maximum age, and removed (along with its ${RESULTS} and ${USERS} hierarchy
+links.
+
+"""
+
+import sys
+import os
+import re
+import shutil
+from datetime import datetime
+from argparse import ArgumentParser
+from configparser import ConfigParser, NoOptionError
+
+from pbench import (
+    PbenchConfig,
+    BadConfig,
+    get_pbench_logger,
+)
+from pbench.indexer import _STD_DATETIME_FMT
+
+# from pbench.report import Report
```

Comment:
remove

Reply:
Actually, this is a place holder for a report that I have not implemented ...

Reply:
Latest commit now generates a report.